### PR TITLE
feat(textarea, timepicker, select): improvements in textarea, select,…

### PR DESCRIPTION
**Pull request by shravan-balasubramanian**
… timepicker
Solves #94 

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

File: src/components/select/select.scss [hunk 1]
```diff
@@ -205,6 +205,10 @@ label {
   .dropdown-status-icon.expanded {
     transform: rotate(315deg);
   }
+
+  &.select-disabled {
```
Comment (asifahmedfw) [inline src/components/select/select.scss:209]: Add some css and use not-allowed cursor symbol.
Refer the DSM.

File: src/components/select/select.tsx [hunk 2]
```diff
@@ -87,11 +87,18 @@ export class Select {
   }
 
   private innerOnFocus = (e: Event) => {
+    if (this.disabled) {
```
Comment (asifahmedfw) [inline src/components/select/select.tsx:90]:
```suggestion
    if (!this.disabled) {
```
Avoid return.

File: src/components/textarea/textarea.scss [hunk 3]
```diff
@@ -75,8 +76,10 @@ label {
     font-weight: var(--font-weight-500);
     letter-spacing: 0;
     line-height: 20px;
+    font-family: inherit;
     cursor: text;
     display: inline-block;
+    color: $app-primary-black;
```
Comment (asifahmedfw) [inline src/components/textarea/textarea.scss:82]: Can we not use Sass variables wherever possible. You can use the freshworks colors. We plan to slowly remove this.

File: src/components/select/select.tsx [hunk 4]
```diff
@@ -240,10 +247,11 @@ export class Select {
         }}
       >
         {this.label !== '' ? <label class={{ 'required': this.required }}> {this.label} </label> : ''}
-        <div class="select-container">
+        <div class={{ 'select-container': true }}>
```
Comment (asifahmedfw) [inline src/components/select/select.tsx:250]: Should we have conditional classes for single items?

File: src/components/select/select.stories.mdx [hunk 5]
```diff
@@ -34,6 +34,30 @@ import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
...
+    {() => `
+    <fw-select label="Select the house" required="true" value="1" disabled>
+        <fw-select-option value="1">Starks</fw-select-option>
+        <fw-select-option value="2">Lannisters</fw-select-option>
+    </fw-select>
+    `}
+  </Story>
+</Preview>
+
+### Readonly
```
Comment (asifahmedfw) [inline src/components/select/select.stories.mdx:49]: Can we remove this?

Comment (shravan-balasubramanian) [reply src/components/select/select.stories.mdx:49]: Date picker depends on this

**Pull request merged**